### ISSUE TITLE
Does this multiselect contain these features?

### ISSUE THREAD

**Issue by AmitMY** [bentorfs/angular-bootstrap-multiselect#12]
Hey, this is not an issue, but becasue the documentation is not the most descriptive of its parameters, I would like to know if this support several things:
- ng-change - on change, can I trigger a function? (example: ng-change="myFunction()"
- paramter-key - If my array is of objects, can I deside what parameter of the object is the key? (example parameter-key="custon_id")
- parameter-value - format the shown value (example: parameter-value="name - key: id". When the object is {name: "example", key: "ex", id: 1} will show "example - ex: 1")
- button-title - custom button title for the button to show more than 1 word. (example: button-title="Countries")

Besides these, I realy like the design, and options for the directive, and looking forward using it if above things exist (I realy dont need everything, most I can simulate on my own, but I want to know if some exist for I not to write jumbled code inside your neat directive)

Thanks

**Comment by bentorfs** [comment]
Hi there,

Apologies if these answers are not super-clear, but I haven't worked on
this project (and angularJS in general) for quite a while.
- ng-change: Unless it is supported out of the box by ngmodelcontroller,
  this is not supported. Should be easy to implement though!
- parameter key: you can use the "id-prop" attribute for that, which is
  described in the documentation (
  http://bentorfs.github.io/angular-bootstrap-multiselect/) in section
  (objects as model).
- Parameter value: you can use "display-prop" attribute for that
  (described in the same place in the docs). You can not use a combination of
  values though.
- Button title: you can use "default-text" attribute for this. It's not
  documented though. It was added by someone else in a merge request I
  approved.

Thanks for your kind words. If you add any features, please make a merge
request for it, so that others can benefit! Also, if the documentation is
lacking for you, I'll approve merge requests. I'm not very actively
maintaining this project :)

**Comment by AmitMY** [comment]
Thanks.
I created what I wanted except for the full on editable display prop, this might happen later. (At the moment I want to use this directive, but am not sure if will)

This is the first time I "contribute" to a project, and I am not sure if I am doing something wrong.
I commited the changes, and created a markdown file specifieng what I change and why.

I also created the pull request, hope that was not wrong as well.

Thanks again for this project, you have a really compact and nice code to work with!
